### ISSUE TITLE
Person schema and Profile API

### ISSUE THREAD

**Issue by edowado** [beakerbrowser/unwalled.garden#44]
Does it make sense on writing a ton of data into the person schema ? (or it must contain just the type, description and title fields?)
Also how can I add data to the profile (dat.json), its a simple write to a json file or it ll be like the POSTS API (await posts.add({body, visibility})) ?

**Comment by pfrazee** [comment]
Yeah the profile information is currently only "title", "description", and an avatar. I think we should consider adding a profile/resume schema, perhaps https://jsonresume.org/schema/
